src/sonic-build-hooks/scripts/post_run_buildinfo

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
+#set -x
+
+IMAGENAME=$1
 
 . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
 
 # Collect the version files
 collect_version_files $POST_VERSION_PATH
 
-[ -d $BUILD_VERSION_PATH ] && [ ! -z "$(ls -A $BUILD_VERSION_PATH)" ] && cp -rf $BUILD_VERSION_PATH/* $POST_VERSION_PATH
-rm -rf $BUILD_VERSION_PATH/*
+	
```

Comment:
Additional space characters can be removed.

Reply:
done